src/Client.js

```diff
@@ -1739,6 +1739,24 @@ class Client extends EventEmitter {
         }
         return 0;
     }
+
+    /**
+     * Sync history conversation
+     * @param {string} chatId
+     * @return {boolean} true/false
+     */
+    async syncHistory(chatId) {
+        return await this.pupPage.evaluate(async (chatId) => {
```

Comment:
We don't need to use `await` here since the method already returns `Promise`.
```suggestion
        return this.pupPage.evaluate(async (chatId) => {
```